lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/process/RunnableProcess.java

```diff
@@ -36,7 +36,7 @@
 /**
  * A runnable class that encapsulates a process.
  */
-public final class RunnableProcess implements Runnable {
+public class RunnableProcess implements Runnable {
```

Comment:
It was to extend this class, but looking at it again, it seems unnecessary to do so. I'll put it back up